src/main/java/it/uniroma2/pellegrini/z64sim/util/sys/OS.java

```diff
@@ -0,0 +1,123 @@
+package it.uniroma2.pellegrini.z64sim.util.sys;
```

Comment:
We use REUSE compliance for license management in this project. Every file should have a header describing the licensing and the copyright owner. See [here](https://github.com/alessandropellegrini/z64sim/blob/develop/src/main/java/it/uniroma2/pellegrini/z64sim/z64sim.java#L1-L4) for example.

Feel free to take ownership of this code, and apply the license you find fit.